### PR TITLE
Pure Number input and selection list

### PR DESCRIPTION
**Pull request by genusistimelord**
I have updated the number input into a pure version.

### CODE REVIEW

**Comment by genusistimelord** [comment]
@Kaiden42 I await you on this pull request before i do more as i have no idea which ones you might of done by now.

File: examples/pure/number_input/Cargo.toml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+[dependencies]
+iced_aw = { path = "../../..", default-features = false, features = [
+    "number_input",
+    "icons",
+    "pure",
+] }
+
+[dependencies.iced]
+git = "https://github.com/iced-rs/iced.git"
+#branch = "generic-widget-tree"
```
Comment (Kaiden42) [inline examples/pure/number_input/Cargo.toml:19]: Iced in your example is not up to date with the rev used in the rest of the project.

File: examples/pure/selection_list/Cargo.toml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    "selection_list",
+    "pure"
+] }
+
+[dependencies.iced]
+git = "https://github.com/iced-rs/iced.git"
+rev = "ba33e92818ec181508b4957d1eae5b2beb9ea221"
+#branch = "generic-widget-tree"
+#rev = "af7cbf0"
+version = "0.4"
```
Comment (Kaiden42) [inline examples/pure/selection_list/Cargo.toml]: Maybe it's better to comment-out that line of code :wink:

**Review by Kaiden42 — APPROVED** [review]
Thank you very much! :+1:

**Pull request merged**